Postman/PostSeries.js

```diff
@@ -0,0 +1,148 @@
+import Promise from 'bluebird';
+import UserModel from 'models/User';
+import BookModel from 'models/Book';
+import Mailer from 'common/Mailer';
+import SeriesModel from 'models/Series';
+
+function filterNewReleases(src, dist, props) {
+  /* Series.currentContainsとSeries.recentContainsの差分配列を生成 */
+  return dist.filter((item)=> {
+    let isDiff = true;
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).